src/main/blackbox/blackbox.c

```diff
@@ -1402,8 +1403,12 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_FEEDFORWARD_JITTER_FACTOR, "%d",  currentPidProfile->feedforward_jitter_factor);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_FEEDFORWARD_BOOST, "%d",          currentPidProfile->feedforward_boost);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_FEEDFORWARD_MAX_RATE_LIMIT, "%d", currentPidProfile->feedforward_max_rate_limit);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ANGLE_FEEDFORWARD, "%d",          currentPidProfile->pid[PID_LEVEL].F);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_LEVEL_GYRO_FF_SCALE, "%d",        currentPidProfile->levelGyroFeedforwardScale);
 #endif
-
+        BLACKBOX_PRINT_HEADER_LINE("angle_dterm_lpf_hz", "%d",              currentPidProfile->angle_dterm_lpf_hz);     
```

Comment:
Please use `PARAM_NAME` format for all angle and horizon mode CLI entries.

Reply:
Agreed. Once functionality is reviewed and approved I'll do a full renaming before merge.